app/controllers/users_controller.rb

```diff
@@ -1,4 +1,11 @@
 class UsersController < ApplicationController
-  def new
+  def create
+    user = User.create()
   end
+
+  private
+  def user_params
+    params.permit(:username, :password)
+  end
+  
```

Comment:
I don't think this controller does anything right now, where are we hooking into it?

Reply:
It doesn't do anything yet, I'm currently working on wiring up the signup to this